### ISSUE TITLE
ReactPerf throws production is not defined error

### ISSUE THREAD

**Issue by iroy2000** [facebook/react#7032]
I'm using webpack to build my react project ( "react": "^0.14.6" ), and somehow it is throwing production is not defined ( ReferenceError ) in ReactPerf.    Because in my webpack setting, I was injecting the NODE_ENV ( which was working fine before ), any advice on that ?

![screen shot 2016-06-13 at 11 11 05 am](https://cloud.githubusercontent.com/assets/487520/16018124/ddd1da32-3157-11e6-84ea-78e1ee1ec54b.png)

![screen shot 2016-06-13 at 11 11 05 am]
(https://cloud.githubusercontent.com/assets/487520/16018095/be628a98-3157-11e6-86dc-0cb09abd7bb9.png)

**Comment by gaearon** [comment]
`DefinePlugin` just substitutes what you specify as the value. You are specifying `'production'`, so it embeds the contents of the string (`production`) in your code, and it blows up. The correct configuration for that key is `NODE_ENV: "'production'"` (notice the apostrophes inside the quotes!) This way you Webpack will put a JS string in your code, and it will work.

You’ll notice that most example configurations use `{ NODE_ENV: JSON.stringify('production') }`. This is exactly the reason why they’re doing it. `JSON.stringify('production')` becomes a string containing `"production"`. But you can specify it in the raw `'"production"'` or `"'production'"` form if you’d like.

**TLDR:** Replace `NODE_ENV: "production"` with `NODE_ENV: "'production'"` in your config.
